plugins/linkproviders/youtube.go

```diff
@@ -0,0 +1,158 @@
+package linkproviders
+
+import (
+	"net/url"
+	"net/http"
+	"encoding/json"
+	"strings"
+	"fmt"
+
+	"github.com/ChannelMeter/iso8601duration"
```

Comment:
Can we explicitly import this as "duration"? Took me a minute to find where this was used.

```
duration "github.com/ChannelMeter/iso8601duration"
```